satpy/modifiers/atmosphere.py

```diff
@@ -149,8 +150,8 @@ def __call__(self, projectables, optional_datasets=None, **info):
                                             band.attrs['sensor'])
 
         atm_corr = corrector.get_correction(satz, band.attrs['name'], band)
-        proj = band - atm_corr
-        proj.attrs = band.attrs
+        proj = xr.DataArray(atm_corr, attrs=band.attrs,
```

Comment:
this should be `band - atm_corr`, right?

Reply:
That's the main point being addressed here. From the PR description:

> The data returned by the pyspectral routine is the aleady corrected version, not a diffence/offset that has to be substracted from the input data. This PR should fix this.

Reply:
my bad, sorry.